resources/routes/ProtectedRoute.tsx

```diff
@@ -4,20 +4,18 @@
 
 import React from 'react';
 import { Navigate } from 'react-router-dom';
-import { useDispatch, useSelector } from 'react-redux';
-import { RootState } from '../store/combined';
+import { sessionStore } from '@store/sessionStore';
 
 export interface ProtectedRouteProps {
-    children : React.ReactNode;
+    children: React.ReactNode;
     auth: boolean;
 }
 
 export default function ProtectedRoute(props: ProtectedRouteProps) {
-    const dispatch = useDispatch();
-    const getToken = useSelector((state: RootState) => state.tokenReducer);
+    const token = sessionStore((state) => state.token)
     return (
         <>
-            { props.auth ? getToken != '' ? props.children : (<Navigate to="/login" />) : props.children }
+            {props.auth ? token != '' ? props.children : (<Navigate to="/login" />) : props.children}
```

Comment:
Codacy has a fix for the issue: [Replace `············{props.auth·?·token·!=·''·?·props.children·:·(<Navigate·to="/login"·/>)·:·props.children}` with `······{props.auth·?·(⏎········token·!=·""·?·(⏎··········props.children⏎········)·:·(⏎··········<Navigate·to="/login"·/>⏎········)⏎······)·:·(⏎········props.children`](https://app.codacy.com/gh/bluespada/go-telescope/pullRequest?prid=9653145)
```suggestion
      {props.auth ? (
```